llvm/utils/UpdateTestChecks/common.py

```diff
@@ -196,6 +196,13 @@ def __call__(self, parser, namespace, values, option_string=None):
         default=[],
         help="List of regular expressions such that, for matching global value declarations, literal integer values should be encoded in hex in the associated FileCheck directives",
     )
+    parser.add_argument(
+        "--skip-check-label",
+        action="store_true",
+        dest="skip_check_label",
+        default=False,
+        help="Activate CHECK-LABEL line generation per function",
```

Comment:
Shouldn't it be "disable CHECK-LABEL" line generation?

Reply:
That's correct. Thanks.